Created Cancel Free From Invite method (#59)
signnow/SignNowNodeSDK: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [v1.6.0] - Unreleased Phase 5

### Added

> - Implemented *Cancel Free Fom Invite* method with documentation and example i [README](https://github.com/signnow/SignNowNodeSDK/blob/master/README.md#cancel-freeform-invite)

Reviewer: ```suggestion
- Implemented *Cancel Free Form Invite* feature with documentation and example in [README](https://github.com/signnow/SignNowNodeSDK/blob/master/README.md#cancel-freeform-invite)
```

> - Created [Applet](https://github.com/signnow/SignNowNodeSDK/blob/master/samples/applets/cancel-freeform-invite.js)

Reviewer: ```suggestion
- Created CLI [Applet](https://github.com/signnow/SignNowNodeSDK/blob/master/samples/applets/cancel-freeform-invite.js) for *Cancel Free Form Invite* feature
```
